stubs/google/cloud/tasks_v2/types.pyi

```diff
@@ -0,0 +1,10 @@
+from typing import Any
+from google import protobuf
+
+
+class Task:
+    name: str = ...
+
+class target_pb2(protobuf.message.Message):
+    class HttpMethod(protobuf.message.Message):
```

Comment:
Isn't this more like subclass of `Enum`?

Reply:
Not exactly enum because enums don't return a variable of type str right? Like target_pb2.HttpMethod.POST will be enum object not string right?
I noticed we don't need protobuf.message.Message here. Used by mistake. Thanks!